nva-file-model/src/test/java/no/unit/nva/file/model/FileModelTest.java

```diff
@@ -75,6 +80,14 @@ void shouldReturnEmptySetWhenLicenseLabelsAreNull() {
         assertThat(license.getLabels(), is(anEmptyMap()));
     }
 
+    @Test
+    void shouldReturnSerializedModel() throws JsonProcessingException {
```

Comment:
add an assertion that FileSet has not empty values, In that way we know* that all fields are serialized and deserialized.

*"know" here assumes that equals checks all object properties.